user-guide/Cloud_Platform/Catalog/Looking_up_a_catalog_item.md

```diff
@@ -0,0 +1,23 @@
+---
+uid: Looking_up_an_item_in_the_catalog
+---
+
+# Looking up an item in the Catalog
+
+When you open up the Catalog module, a search box is displayed.
+
+1. In the search box, enter (part of) the name of the item you are looking for (e.g. a connector, a package, etc.).
+
+    As you type, a list with the top 5 results will be displayed under the search box.
+
+1. As soon as you notice the name of the item you are looking for, click it and continue to the next step.
+
+    If you cannot find the item you are looking for in this list you can choose to get a list of all the search results, by clicking the . This will open up the browse page. On the left side of this page you have filters that allow you to narrow down your search.
```

Comment:
"... by clicking the . This will open up..." There is probably a word missing there?